rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -624,9 +624,9 @@ def predict_label(self, message):
             if X.any() and label_ids.size > 0:
 
                 # normalise scores if turned on
-                if self.loss_type == "softmax" and self.normalize_top_num_intents > 0:
-                    label_ids = label_ids[: self.normalize_top_num_intents]
-                    message_sim = message_sim[: self.normalize_top_num_intents]
+                if self.loss_type == "softmax" and self.ranking_length > 0:
```

Comment:
if they put `self.ranking_length = 50` do we want to output 50 intents?

Reply:
see https://github.com/RasaHQ/rasa/pull/4902#discussion_r358868828